teuthology/nuke.py

```diff
@@ -400,19 +400,24 @@ def check_console(hostname):
         host=shortname,
         domain=config['ipmi_domain'])
     log.info('checking console status of %s' % cname)
-    if not console.check_status():
-        # not powered on or can't get IPMI status.  Try to power on
-        console.power_on()
```

Comment:
it's just behavior I wouldn't expect, and would in general prefer to enable with a switch.  but it's more about surprise anyway.  carry on with the Large Hammer Nuke.